projects/boringssl/build.sh

```diff
@@ -48,3 +46,20 @@ for F in $fuzzerFiles; do
     zip -j $OUT/${fuzzerName}_seed_corpus.zip $SRC/boringssl/fuzz/${fuzzerName}_corpus/*
   fi
 done
+
+rm -rf genfiles && mkdir genfiles && $SRC/LPM/external.protobuf/bin/protoc asn1_pdu.proto --cpp_out=genfiles --proto_path=$SRC/fuzzing/proto/asn1-pdu-proto/
+
+fuzzerLPMFiles=$(find $SRC/ -maxdepth 1 -name "*.cc")
+
+for F in $fuzzerLPMFiles
+do
+  fuzzerName=$(basename $F .cc)
+  echo "Building fuzzer $fuzzerName"
+  $CXX $CXXFLAGS -I genfiles -I . -I $SRC/libprotobuf-mutator/ -I $SRC/LPM/external.protobuf/include -I include $LIB_FUZZING_ENGINE \
+      $F genfiles/asn1_pdu.pb.cc $SRC/fuzzing/proto/asn1-pdu-proto/asn1_proto_to_der.cc \
+      -I $SRC/boringssl/include ./ssl/libssl.a ./crypto/libcrypto.a \
```

Comment:
```suggestion
      -I $SRC/boringssl/include 
      $F genfiles/asn1_pdu.pb.cc $SRC/fuzzing/proto/asn1-pdu-proto/asn1_proto_to_der.cc \
      ./ssl/libssl.a ./crypto/libcrypto.a \
```

It looks like you may have added the -I to get things working, but let's try to keep them grouped together.

I'm also not sure whether we should order the files differently (e.g. `.a`, support files, your `$F`), but I suspect @Dor1s might have an idiomatic/common pattern.

Reply:
It doesn't really matter for us as long as things are working. We have too many projects with different build scripts so there are no standards for those :)

Reply:
Hmm, doing that broke it `clang-11: error: no input files`. I followed the existing build structure when writing this https://github.com/google/oss-fuzz/blob/master/projects/boringssl/build.sh#L45.

Reply:
that's because Ryan's suggestion lacks a `\` on line 59 I think, it's just a typo, try adding that slash to make sure the command is continued on the next line

Reply:
oh of course. I missed that somehow xD